src/components/card/z-card-info/index.tsx

```diff
@@ -61,18 +69,48 @@ export class zCardInfo {
 
   renderGeneralSection() {
     return (
-      <section>
-        Autore: <b>{this?.cardData?.author}</b>
-        <br />
-        Anno: <b>{this?.cardData?.year}</b>
-        <br />
+      <section
+        class={`info-wrapper ${this.hiddenContent ? "hidden" : ""}`}
+        onClick={() => {
+          if (this.hiddenContent) this.tooltip = !this.tooltip;
+        }}
+        ref={(el) => (this.infoWrapper = el)}
+      >
+        {this?.cardData?.author ? (
+          <span>
+            Autore: <b>{this.cardData.author}</b>
+            <br />
+          </span>
+        ) : (
+          ""
+        )}
+        {this?.cardData?.year ? (
+          <span>
+            Anno: <b>{this.cardData.year}</b>
+            <br />
+          </span>
+        ) : (
+          ""
+        )}
         {this?.cardData?.title}
         <br />
         {this?.cardData?.description}
       </section>
     );
   }
 
+  renderTooltip() {
+    if (!this.tooltip) return;
+
+    return (
+      <z-tooltip
+        content={`${this?.cardData?.title} ${this?.cardData?.year} ${this?.cardData?.author} ${this?.cardData?.description}`}
```

Comment:
const {title, year, author, description} = this.cardData!;
[...]
content={`${title}...}